tests/Unit/CssInlinerTest.php

```diff
@@ -3467,86 +3461,53 @@ public function inlineCssNotCopiesInlinableRuleAfterImportRuleToStyleElement():
      */
     public function provideValidAtRulesWithSurroundingCss(): array
     {
-        $datasets = [];
-
-        foreach (self::BLACK_BOX_AT_RULES as $name => $rule) {
-            $datasets += [
-                'single ' . $name => [
-                    'before' => '',
```

Comment:
`DataProviders::cross()` throws an exception if the dataset items have string keys, so unfortunately the descriptive keys can't be retained.

Reply:
@JakeQZ I'm on it! <3